planet/api/orders.py

```diff
@@ -74,7 +75,9 @@ def __init__(
                 base url.
         """
         self._session = session
-
+        self.NUM_DOWNLOADS = 0
```

Comment:
two thoughts here:
1. I suggest using lowercase for variables like this since they are not global vars
2. I'm curious about tracking downloads/skips at this level. What happens if we use the client to download multiple orders? What if the client is downloading multiple orders async? Multiple sources would be changing and accessing these values. Seems like something we want just returned by the functions themselves to avoid confusion.

Reply:
1. Done!
2. Good catch - thank you! You’re right, I never reset `NUM_DOWNLOADS` and `NUM_DOWNLOADS_SKIPPED` both back to 0. I think you’re right, it would make more sense to have `download_asset()` return these two values, instead of storing them in the class object, `self`.